docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:xenial
+
+RUN apt-get update \
+    && apt-get install -y \
+        curl \
+        lsb-release \
+    && curl -sS https://dl.google.com/cloudagents/install-logging-agent.sh | bash \
+    && /opt/google-fluentd/embedded/bin/gem uninstall fluent-plugin-google-cloud \
+    && /opt/google-fluentd/embedded/bin/gem install fluent-plugin-google-cloud -v 0.6.8 \
```

Comment:
Do we want to just run `gem install` here and let it pick up the latest released gem version? Or is there a good reason to freeze the version?

Reply:
In fact, I'm thinking about bumping the `fluent-plugin-google-cloud` version to `0.6.8` in `google-fluentd` and release a new package some time next week. After that we don't need the `gem uninstall` and `gem install` stuff.

WDYT?

Reply:
I think the decision we need to make is whether we're tracking the latest released package, or the latest released gem. The latter would let us push the container image ahead of the package, but I'm not sure how much that buys us. It seems that you're leaning towards the former.

Reply:
Yeah, I'm leaning towards the former. If the user really wants, they can always build their custom image and bump the gem version there.

WDYT?

Reply:
SGTM. So let's remove this from here and release 1.5.19 instead?

Reply:
Removed.